staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -718,6 +718,10 @@ type KubeletConfiguration struct {
 	// Default: ["pods"]
 	// +optional
 	EnforceNodeAllocatable []string `json:"enforceNodeAllocatable,omitempty"`
+	// Whitelist of unsafe sysctls or sysctl patterns (ending in *). Use these at your own risk.
```

Comment:
The `use these at your own risk` is unusual wording for our API doc.  I might suggest describing why these might be unsafe and giving an example or two (now that these are moving into formal API).

Describe via example how multiple patterns work (add a sentence 'For instance, providing pattern blah* and sysctl other will allow pods to set any sysctl that has the prefix blah or the exact sysctl name other')

Reply:
Also include a sentence describing the default value `By default, no unsafe sysctls will be allowed.`

Reply:
fixed